src/backend/views/pages/index.twig

```diff
@@ -14,11 +13,13 @@
 </script>
 <body class="landing-body">
   {% include "components/header.twig" %}
-  <div class="landing-loader" id="frame-loader">
-    {{ svg('loader') }}
+  <div class="landing-content">
```

Comment:
now that there's no landing inserted via iframe, maybe we should use different class names?